services/src/main/scala/cromwell/services/metadata/MetadataService.scala

```diff
@@ -198,39 +198,55 @@ object MetadataService {
   final case class WorkflowQueryFailure(reason: Throwable) extends MetadataQueryResponse
 
   implicit private class EnhancedWomTraversable(val womValues: Iterable[WomValue]) extends AnyVal {
-    def toEvents(metadataKey: MetadataKey): List[MetadataEvent] = if (womValues.isEmpty) {
+    def toEvents(metadataKey: MetadataKey, fileMap: Map[Path, Path]): List[MetadataEvent] = if (womValues.isEmpty) {
```

Comment:
Would be good to add some comments here explaining the purpose of the new file map.

Reply:
I added a comment to `womValueToMetadataEvents`, which probably what people will look at first over this private helper class. Does that satisfy your request?

Reply:
It does, thanks!